src/main/scala/com.snowplowanalytics/forex/ForexConfig.scala

```diff
@@ -21,18 +23,37 @@ object EodRoundDown extends EodRounding
 /** Round to next day*/
 object EodRoundUp extends EodRounding
 
+/**
+ * There are three types of accounts supported by OER API.
+ * For scala-forex library, the main difference between Unlimited/Enterprise
+ * and Developer users is that users with Unlimited/Enterprise accounts
+ * can use the base currency for API requests, but this library will provide
+ * automatic conversions between OER default base currencies(USD)
+ * and user-defined base currencies. However this will increase calls to the API
+ * and will slow down the performance.
+ */
+sealed trait AccountType
```

Comment:
I'd move this inside `ForexConfig` companion objects and all case objects into `AccountType` companion object as well to follow ADT convention.